### ISSUE TITLE
Go1.6 compatibility

### ISSUE THREAD

**Issue by ka2n** [moovweb/gokogiri#91]
```bash
$ git rev-parse HEAD
eafc181356de1da75468a7bdf54b430393f9c0e3

$ go version
go version go1.6 darwin/amd64

$ go test
--- FAIL: TestParseHtml (0.00s)
panic: runtime error: cgo argument has Go pointer to Go pointer [recovered]
        panic: runtime error: cgo argument has Go pointer to Go pointer

goroutine 19 [running]:
panic(0x41b77a0, 0xc8200605a0)
        /usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
testing.tRunner.func1(0xc8200a2000)
        /usr/local/Cellar/go/1.6/libexec/src/testing/testing.go:467 +0x192
panic(0x41b77a0, 0xc8200605a0)
        /usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:426 +0x4e9
github.com/moovweb/gokogiri/xml._cgoCheckPointer0(0x4160ac0, 0xc820064300, 0x0, 0x0, 0x0, 0xc820078030)
        ??:0 +0x4d
github.com/moovweb/gokogiri/xml.(*XmlNode).serialize(0xc82007a080, 0x41, 0xc820060570, 0x6, 0x10, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /Users/ka2n/src/github.com/moovweb/gokogiri/xml/node.go:773 +0x172
github.com/moovweb/gokogiri/xml.(*XmlNode).ToHtml(0xc82007a080, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /Users/ka2n/src/github.com/moovweb/gokogiri/xml/node.go:826 +0x90
github.com/moovweb/gokogiri/xml.(*XmlNode).ToBuffer(0xc82007a080, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /Users/ka2n/src/github.com/moovweb/gokogiri/xml/node.go:833 +0xc7
github.com/moovweb/gokogiri/xml.(*XmlNode).String(0xc82007a080, 0x0, 0x0)
        /Users/ka2n/src/github.com/moovweb/gokogiri/xml/node.go:841 +0x42
github.com/moovweb/gokogiri.TestParseHtml(0xc8200a2000)
        /Users/ka2n/src/github.com/moovweb/gokogiri/gokogiri_test.go:18 +0x29f
testing.tRunner(0xc8200a2000, 0x4320ba0)
        /usr/local/Cellar/go/1.6/libexec/src/testing/testing.go:473 +0x98
created by testing.RunTests
        /usr/local/Cellar/go/1.6/libexec/src/testing/testing.go:582 +0x892
exit status 2
FAIL    github.com/moovweb/gokogiri     0.015s

$ GODEBUG=cgocheck=0 go test
PASS
ok      github.com/moovweb/gokogiri     0.011s
```

**Comment by jbowtie** [comment]
I'm pretty sure this is the same bug fixed in #89

**Comment by ka2n** [comment]
@jbowtie Thank you for letting me know.
